Automattic/wp-calypso
# Editor Stats - Fix `from_template_selector` property for insert block events.

#### Changes proposed in this Pull Request
Fixes https://github.com/Automattic/wp-calypso/issues/53637
Updates the filter used to populated `from_template_selector` property on block inserted events to match the string id used in the start page templates component.  

#### Testing instructions



* Follow steps to enable tracking debugging at PCYsg-nrf-p2
* Create a new page and select a page layout to insert.
* After the blocks are inserted, verify the `wpcom_block_inserted` events correctly show the `from_template_selector` property as `true`.
